app/portainer/helpers/stackHelper.js

```diff
@@ -22,6 +22,24 @@ angular.module('portainer.app').factory('StackHelper', [
       );
     }
 
+    function findDeepAll(obj, key) {
```

Comment:
@MaximeBajeux Nice helper, could we make it available in a more general helper as it could be used elsewhere?